dependencies_with_url.csv

```diff
@@ -146,6 +146,7 @@ com.codahale.metrics:metrics-core:jar:3.0.2:compile,MIT,https://github.com/codah
 com.codahale.metrics:metrics-graphite:jar:3.0.2:compile,MIT,https://github.com/codahale/metrics
 com.esotericsoftware.reflectasm:reflectasm:jar:shaded:1.07:compile,BSD,https://github.com/EsotericSoftware/reflectasm
 com.fasterxml.jackson.core:jackson-annotations:jar:2.2.3:compile,ASLv2,http://wiki.fasterxml.com/JacksonHome
+com.fasterxml.jackson.core:jackson-annotations:jar:2.7.0:compile,ASLv2,http://wiki.fasterxml.com/JacksonHome
```

Comment:
Maybe this is the cause [of the issue](https://github.com/apache/metron/pull/1458#issuecomment-516178822) @merrimanr found during testing which I was not able to replicate.  I will track this down.

Reply:
Actually, jackson-databind:2.7.4, which is the version defined by $global_jackson_version pulls in jackson-annotations:2.7.0, so this seems to be necessary.  

I had to move the jackson-databind dependency up so as to not conflict. I am not sure why the license validator did not pick-up on this before.
```
@ ~/Development/metron/metron-analytics/metron-profiler-storm
$ mvn dependency:tree
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building metron-profiler-storm 0.7.2
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ metron-profiler-storm ---
[INFO] org.apache.metron:metron-profiler-storm:jar:0.7.2
[INFO] +- com.fasterxml.jackson.core:jackson-databind:jar:2.7.4:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.7.0:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.7.4:compile

Reply:
Actually, I think I can also explicitly pull-in `jackson-annotations:2.7.4` in the pom to avoid pulling in `jackson-annotations:2.7.0` as a transitive. That is what the other projects seem to do like `metron-parsers-common`.

That will avoid pulling in some other random version like 2.7.0.  I have no idea why a transitive for jackson-databind:2.7.4 would be jackson-annotations:2.7.0.

Reply:
See 1ef49cfec.